sklearn/metrics/tests/test_score_objects.py

```diff
@@ -247,6 +247,31 @@ def test_regression_scorers():
     assert_almost_equal(score1, score2)
 
 
+def test_log_loss():
+   
+
+
+
+    X = [[1,1], [1,1], [2, 2], [2, 2]]
+    y = [1, 1, 2, 2]
+    clf = DecisionTreeClassifier()
+    clf.fit(X, y)
+
+
+
+    y_score = clf.predict_proba([[2,2], [2,2]])
+    y_true = [2,2]
+    expected_value = np.log(1)*2
+
+    # y_true are the same, not use labels arg, you will get error 
+    error_logloss = log_loss(y_true, y_score)
+    assert_not_equal(expected_value, error_logloss)
```

Comment:
It might be better to test the current_behavior, where if labels is not present, for a single label case, it is encoded as the negative class, which means the log loss will be `-np.mean(np.log(y_score[:, 0]))`

rather than test that the "current behaviour" is wrong